lib/diagnostic.rb

```diff
@@ -106,10 +106,12 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = {first_name:"Wei",age:100}'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
```

Comment:
I think you added the does not have the #{key} bit from a previous talk, so good job using your resources! 

That said there's a few issues there. I think you want something more like this : 

'person.default = "#{person[:first_name]} <stuff here>"'